getting_started/first_2d_game/03.coding_the_player.rst

```diff
@@ -139,8 +139,32 @@ First, we need to check for input - is the player pressing a key? For this game,
 we have 4 direction inputs to check. Input actions are defined in the Project
 Settings under "Input Map". Here, you can define custom events and assign
 different keys, mouse events, or other inputs to them. For this game, we will
-just use the default events called "ui_right" etc that are assigned to the arrow
-keys on the keyboard.
+map the arrow keys to the four directions.
+
+Click on *Project -> Project Settings* to open the project settings and then
+click on the *Input Map* tab at the top. Enter "move_right" and then click
+the "Add" button to add the ``move_right`` action, as shown here:
+
+.. image:: img/input-mapping-add-action.png
+
+Now we need to add a key assignment to this action. Press the "+" button and
+then press "Key" in the pop-up menu.  A dialog will ask you to press the key
+you want mapped.  Press the right arrow and then click "Ok":
```

Comment:
```suggestion
We need to assign a key to this action. Click the "+" icon on the right, then
click the "Key" option in the drop-down menu. A dialog asks you to type in the
desired key. Press the right arrow on your keyboard and click "Ok".
```

Reply:
Ugh, I tried to be careful about the words I used.  I'll get this right eventually.  😆